build_web_compilers/lib/src/dev_compiler_stack_trace/source_map_stack_trace.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Standalone utility that manages loading source maps for all Dart scripts
+/// on the page compiled with DDC.
+///
+/// This is forked from the equivalent file that ships with the SDK which is
+/// intended for use within bazel only.
+
+import 'package:path/path.dart' as p;
+import 'package:source_maps/source_maps.dart';
+import 'package:stack_trace/stack_trace.dart';
+
+/// This is a fork of package:source_map_stack_trace, a tool for
```

Comment:
is this duplicating the inoformation on line 8?

Reply:
removed